### DISCUSSION TITLE
FR:  Configuration to support a 'pending block' and/or enforce a block by datetime-stamp

### DISCUSSION THREAD

**Discussion by dsbaha** [northpolesec/santa#171] [Ideas]
Hi.  Basically; I would like to notify the end users that X will be blocked in a future date.
To share my thoughts; maybe if policy can have a new string 'PENDING_BLOCKLIST' to have the Santa notification pop-up display once a day to notify the user before moving into the blocklist, this would be awesome.
An additional follow-on thought; have a date-time stamp where the blocklist item would start being enforced.

**Comment by pmarkowsky** [comment]
This is something we're interested in. The trick to this is getting the notifications right so as not to nag users. I've made a few prototypes, where I jokingly called the policy `allow-notify` where we'd pop up a the rule's custom message in a system notification.

e.g. ![https://img.youtube.com/vi/ri8wzUxHas8/0.jpg](https://img.youtube.com/vi/ri8wzUxHas8/0.jpg)

Though it's not the best experience because of Do Not Disturb and other things that can cause system notifications to not be seen by users.

Re: scheduling rules we have that on the Workshop roadmap along with rules with a TTL.
